docs/js-core-1/week-2/learning-objective.md

```diff
@@ -0,0 +1,30 @@
+---
+id: learning-objectives
+title: Learning Objectives
+sidebar_label: Learning Objectives
+---
+
+_The learning objectives under a given topic may be achieved over the multiple weeks of the module, rather than in the first week the topic is introduced._
```

Comment:
I suggest splitting these up and relating them more clearly to the content you are creating week by week.

Reply:
This comment was already on the page before any changes by me.

Reply:
```suggestion
_The learning objectives under a given topic may be achieved over the multiple weeks of the module, rather than in the first week the topic is introduced._
```
We definitely need to fix this. Thanks so much for making it visible!!